test/commands/github/pull_request/sync_repo_test.rb

```diff
@@ -0,0 +1,374 @@
+require "test_helper"
+
+class Github::PullRequest::SyncRepoTest < ActiveSupport::TestCase
```

Comment:
@iHiD This is probably one of the more important tests, which admittedly is a bit unwieldy.